bundle/compliance/cis_azure/rules/cis_9_4/rule.rego

```diff
@@ -15,5 +15,7 @@ finding = result {
 }
 
 is_client_cert_enabled {
-	data_adapter.properties.clientCertEnabled == true
+	# Benchmark and Azure implementation in remediation checks previous implemented value
+	# Reading the description and rule metadata, we've decided to check the value of the property clientCertMode
+	data_adapter.properties.clientCertMode == "Required"
```

Comment:
Should we also check the `data_adapter.properties.clientCertEnabled` value for completeness/sanity?

Reply:
It is part of the schema of the ARG.
If the value doesn't exist it will be evaluated as false, do we want to have no evaluation in that case?

Reply:
In which case should it not exist?